.eslintrc.js

```diff
@@ -1,11 +1,14 @@
 module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
-    project: 'tsconfig.json',
+    project: 'linter.tsconfig.json',
```

Comment:
This file could be rather called `tsconfig.linter.json` like `tsconfig.build.json` in the root folder. But this is not really an issue, this is not important